rules/aws_cloudtrail_rules/aws_console_login.yml

```diff
@@ -0,0 +1,19 @@
+AnalysisType: rule
+LogTypes:
+    - AWS.CloudTrail
+DisplayName: "AWS Console Login"
+RuleID: "AWS.Console.Login"
+Enabled: true
+CreateAlert: false
+Severity: Low
+Detection:
+  - Key: eventName
+    Condition: Equals
+    Value: ConsoleLogin
+AlertContext:
+  - KeyName: caller_ip
+    KeyValue:
+      KeyPath: p_udm.source.ip
+AlertTitle: "{userIdentity.userName} logged into the AWS Management Console"
+DedupPeriodMinutes: 60
+Threshold: 1
```

Comment:
I don't think that `DedupPeriodMinutes` or `Threshold` do anything when `CreateAlert` is set to `False`, these can probably be omitted for simplicity's sake. @maxrichie5 @sesheffield can keep me honest on that.

Reply:
that's correct. Anytime `CreateAlert` is set to `False` it's considered only a `Signal` and anytime that signal matches, we put it into the datalake. The `DedupPeriodMinutes` and `Threshold` only applies to the alerting side of things